easybuild/scripts/install_eb_dep.sh

```diff
@@ -53,6 +53,13 @@ if [ $? -ne 0 ] && [ ! -z $BACKUP_PKG_URL ]; then
     wget ${BACKUP_PKG_URL} && tar xfz *${PKG_VERSION}.tar.gz
 fi
 set -e
+
+# environment-modules needs a patch to work with Tcl8.6
+if [ x$PKG_NAME == 'xmodules' ]; then
+    wget -O 'modules-tcl8.6.patch' 'https://sourceforge.net/p/modules/feature-requests/_discuss/thread/4502a6c9/9aa5/attachment/environment-modules-tcl86.patch'
```

Comment:
why conditional? The patch fixes it in such a way it works for both 8.5 and 8.6?